From 82337a0bd09cb63fdd61df90482aef5494be5846 Mon Sep 17 00:00:00 2001
From: Ignacio Casal Quinteiro <icq@gnome.org>
Date: Wed, 23 Dec 2015 08:46:06 +0100
Subject: [PATCH] Add missing includes

This happens for developing on linux something that is for windows
---
 libgitg/gitg-platform-support-win32.c | 11 +++++++++++
 1 file changed, 11 insertions(+)

diff --git a/libgitg/gitg-platform-support-win32.c b/libgitg/gitg-platform-support-win32.c
index ca7f725..debb9fc 100644
--- a/libgitg/gitg-platform-support-win32.c
+++ b/libgitg/gitg-platform-support-win32.c
@@ -21,6 +21,17 @@
 
 #include <gio/gwin32inputstream.h>
 
+#define SAVE_DATADIR DATADIR
+#undef DATADIR
+#include <io.h>
+#include <conio.h>
+#ifndef _WIN32_WINNT
+#define _WIN32_WINNT 0x0501
+#endif
+#include <windows.h>
+#define DATADIR SAVE_DATADIR
+#undef SAVE_DATADIR
+
 gboolean
 gitg_platform_support_use_native_window_controls (GdkDisplay *display)
 {
-- 
2.5.0

